Quartus/compileflash.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
+# Cleans cached files, compiles .sof, then programs FPGA
+
+quartus_sh --clean minRISC && \
+quartus_sh --flow compile minRISC && \
```

Comment:
nit: don't need the \ since shells are smart enough to infer that && means the line isn't done yet.